src/less/lightbox-dialog/lightbox-dialog.less

```diff
@@ -25,6 +25,27 @@
     .dialog-footer-content();
 }
 
+.lightbox-dialog__image {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-radius: var(--border-radius-100) var(--border-radius-100) 0 0;
+    height: 218px;
+    position: absolute;
+    width: 100%;
+}
+
+.lightbox-dialog__image + .lightbox-dialog__header > * {
```

Comment:
We should be able to simplify this selector to `.lightbox-dialog--expressive .lightbox-dialog__header > *`.